src/index.html

```diff
@@ -9,6 +9,57 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#home" class="logo">
+        <img src="/images/logo.png" alt="sorry image not available">
+      </a>
+      <nav class="nav">
+        <ul class="navList">
+          <li class="navItem">
+            <a href="#" class="navLink is-active">
+              Apple
+            </a>
+          </li>
+          <li class="navItem">
+            <a href="#" class="navLink">
+              Samsung
+            </a>
+          </li>
```

Comment:
add a blank line between same level elements
```suggestion
          <li class="navItem">
            <a href="#" class="navLink is-active">
              Apple
            </a>
          </li>
          
          <li class="navItem">
            <a href="#" class="navLink">
              Samsung
            </a>
          </li>
```